looker.md

```diff
@@ -39,48 +39,16 @@ and you're ready to build.
 
 ## Release
 
-Define Looker's Nexus repository in your `~/.gradle/init.gradle.kts`
-file:
 
-```kotlin
-allprojects {
-    plugins.withId("maven-publish") {
-        configure<PublishingExtension> {
-            repositories {
-                maven {
-                    name = "lookerNexus"
-                    val baseUrl = "https://nexusrepo.looker.com"
-                    val releasesUrl = "$baseUrl/repository/maven-releases"
-                    val snapshotsUrl = "$baseUrl/repository/maven-snapshots"
-                    val release = !project.version.toString().endsWith("-SNAPSHOT")
-                    // val release = project.hasProperty("release")
-                    url = uri(if (release) releasesUrl else snapshotsUrl)
-                    credentials {
-                        username = "xxx"
-                        password = "xxx"
-                    }
-                }
-            }
-        }
-    }
-}
-```
-
-In the above fragment, replace the values of the `username` and
-`password` properties with the secret credentials.
-
-*NOTE* This fragment *must* be in a file outside of your git sandbox.
-If the file were in the git sandbox, it would be too easy to
-accidentally commit the secret credentials and expose them on a
-public site.
 
 *Read the instructions* in the `looker-release.sh` script, then run it.
 The script will only make local changes.
-You'll have a chance to review them before pushing anything to Nexus or GitHub.
+You'll have a chance to review them before pushing anything to Artifact Registry or GitHub.
+Upload to Artifact Registry by running `looker-upload-artifact-registry.sh` after checking the release looks right.
 
 Each release will have a name like `1.21.1-looker` (if the most
-recent official Calcite release is `1.21`) and have a git tag
-`calcite-1.21.1-looker`.
+recent official Calcite release is `1.38`) and have a git tag
```

Comment:
this seems to imply that the official Calcite release 1.38 is equivalent to our tag calcite-1.38.0-looker and I'm not sure that's true right? 

that tag points to this commit: 
https://github.com/looker-open-source/calcite/commit/000b4d957b84e35ee3724694bea1856e96e8568b
which has this parent: 
https://github.com/looker-open-source/calcite/commit/5562ec1ca6acdc17bf328ef09601ba17889e1de9

a.k.a they're not the same

Reply:
Updated and added comment